projects/mci/src/regen_braking.c

```diff
@@ -0,0 +1,27 @@
+#include "regen_braking.h"
+
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "status.h"
+
+// regen braking state 1:ON - 0:OFF
+static uint8_t s_regen_braking_state;
+
+// Callback function to set the regen braking state
+static StatusCode prv_regen_braking_callback(const CanMessage *msg, void *context,
+                                             CanAckStatus *ack_reply) {
+  CAN_UNPACK_REGEN_BRAKING(msg, &s_regen_braking_state);
+  return STATUS_CODE_OK;
+}
+
+StatusCode regen_braking_init(void) {
+  // Default to enabled
+  s_regen_braking_state = 1;
```

Comment:
Similar to above, it would be nice if we define the states somewhere so we don't have to add comments to make it clear

Reply:
done